### ISSUE TITLE
Pretrained weights

### ISSUE THREAD

**Issue by m7schoen** [HRLTY/TP-GAN#1]
Can you please also share pretrained weights for DeepFace and the whole model?

**Comment by erikgajdos** [comment]
Hello, I have the same question, can you please share pretrained weights? It would be great to have them available.

**Comment by HRLTY** [comment]
Please see updated README. Pre-trained model will come later.

**Comment by eladrich** [comment]
Hi, is there any update as to the expected release of the pre-trained model?

**Comment by HRLTY** [comment]
Maybe we’ll retrain it and release one in June.

Thanks!

Sincerely,
Rui
Rui Huang | M.S. Student
Electrical & Computer Engineering Department | Carnegie Mellon University
Hamerschlag Hall, 5000 Forbes Ave | Pittsburgh PA 15213
www.andrew.cmu.edu/user/ruih2/

**Comment by stormcoral** [comment]
@HRLTY  Hi, can you share the details to train the network? Thanks!

**Comment by SUGE2016** [comment]
is pre-trained model release plan changed, or canceled? @HRLTY thanks

**Comment by iqbalnaved** [comment]
Hi , It's been two months since the declared release date on June. Is there any way it can be released soon ? Thanks.

**Comment by HRLTY** [comment]
Sorry for the delay due to coursework. We'll try to release a retrained model weight in Sep. and clean the code a  little.

**Comment by zychen2016** [comment]
Hello，where is the pre-trained model?

**Comment by dagongji10** [comment]
Is anyone run the model sucessfully? I use the CAS-PEAL-R1 dataset to train the model but the loss is nan. Anyone can help me?

**Comment by chenusc11** [comment]
Hi, Just wondering do you still have a plan to share the pre-trained model? Thx @HRLTY

**Comment by aniskacem** [comment]
Hi, 

Thanks for sharing the codes! Will the Pre-trained model be released or not? Please, provide an answer so that we do not waste our time...

Best,

**Comment by SherrySwift** [comment]
Hi, Thanks for sharing the codes! Could you please share the pre-trained model as well? Thx

**Comment by hegc** [comment]
There is a plan to release one pre-trained model?@HRLTY

**Comment by Jeremy-lf** [comment]
it's too disappointed

**Comment by redaghanem** [comment]
Hi, Thanks for sharing the codes! Could you please share the pre-trained model as well? Thx
